app/controller/issues.php

```diff
@@ -553,6 +601,20 @@ public function copy($f3, $params)
     protected function _saveUpdate()
     {
         $f3 = \Base::instance();
+
+        //remove parent if user has no rights to it
+        $validateUser = $f3->get("user_obj");
+        if (( $f3->get("POST.parent_id")) && !($validateUser->role == 'admin')) {
+            $helper = \Helper\Dashboard::instance();
+            $groupString = implode(",", $helper->getGroupIds()) . "," . $validateUser->id;
+            $searchFilter = " id = " . $f3->get("POST.parent_id") . " AND owner_id IN (". $groupString .")";
```

Comment:
Concatenating unverified user data into the SQL directly is unsafe, I'll edit this to use a parameter instead. Same with line 7077.